src/App/Pages/Accounts/HomePageViewModel.cs

```diff
@@ -107,11 +109,11 @@ public async Task InitAsync()
 
         public void CheckNavigateLoginStep()
         {
-            if (CheckHasRememberedEmail && RememberEmail)
+            if (CheckNavigateToLogin && RememberEmail)
             {
                 StartLoginAction();
             }
-            CheckHasRememberedEmail = false;
+            CheckNavigateToLogin = false;
         }
```

Comment:
I'd change this and the `CheckNavigateToLogin` property and home parameter to something more meaningful to the business like `ShouldCheckRememberEmail` and the method could be `CheckRememberEmail()` or something like that so we focus on the business rules rather than on the outcome (which is navigate to login).
Also it gives the reader a better idea of what the check is about.
What do you think?